tests/automation/message_requests.spec.ts

```diff
@@ -16,79 +16,123 @@ test.beforeEach(beforeAllClean);
 // test.afterEach(() => forceCloseAllWindows(windows));
 // Open two windows and log into 2 separate accounts
 test.describe('Message requests', () => {
-  sessionTestTwoWindows('Message requests accept', async ([windowA, windowB]) => {
-    const [userA, userB] = await Promise.all([newUser(windowA, 'Alice'), newUser(windowB, 'Bob')]);
```

Comment:
any reason we use test.describe for those tests only? Maybe we should just have the sessionTestTwoWindows direclty at the root of the file

Reply:
I think this was initially to run all the Message requests tests at once but i don't think the `describe` function works correctly